### PR TITLE
Ratvar being alive will even more fully empower stuff

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Invoking Nezbere now increases ocular warden damage slightly more, but increases ocular warden range slightly less.
/:cl:

Slabs can be used by anyone, clockwork armor has massive resists and becomes spaceworthy, clockwork treads become noslip, ratvarian spears become permanent, etc.
Structures also have doubled efficiency.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
The round is pretty much already over when ratvar is summoned so i wouldn't feel like this stuff is 100% priority and moreso just a finishing flair.

**Comment by ChangelingRain** [comment]
It is basically just 'hey youve winner, go BE COOL and MURDER PEOPLE'

File: code/game/gamemodes/clock_cult/clock_items/clockwork_armor.dm [hunk 1]
```diff
@@ -10,12 +10,25 @@
...
+	ratvar_act()
 	all_clockwork_objects += src
 
 /obj/item/clothing/head/helmet/clockwork/Destroy()
 	all_clockwork_objects -= src
 	return ..()
 
+/obj/item/clothing/head/helmet/clockwork/ratvar_act()
+	if(ratvar_awakens)
+		armor = list(melee = 100, bullet = 100, laser = 100, energy = 100, bomb = 100, bio = 100, rad = 100, fire = 100, acid = 100)
```
Comment (GunHog) [inline code/game/gamemodes/clock_cult/clock_items/clockwork_armor.dm]: To clarify what this actually does: 90% damage resistance to melee and projectile attacks, immunity to stun projectiles, reduced damage against bombs (and not being gibbed). If you want to make it the ultimate armor, then have it `return FALSE` on `hit_reaction()' during `ratvar_awakens`.

**Comment by ExcessiveUseOfCobblestone** [comment]
my little deathsquad: ratvar is magic

**Pull request merged**